packages/edit-site/src/components/header/index.js

```diff
@@ -109,10 +113,16 @@ export default function Header( { openEntitiesSavedStates } ) {
 				<FullscreenModeClose />
 			</MainDashboardButton.Slot>
 			<div className="edit-site-header__toolbar">
-				<Inserter
-					position="bottom right"
-					showInserterHelpPanel
-					toggleProps={ inserterToggleProps }
+				<Button
+					className="edit-site-header-toolbar__inserter-toggle"
```

Comment:
I can't find any CSS rules targetting this selector.

Reply:
you're right, it's useless.